identity/manager.go

```diff
@@ -11,13 +11,13 @@ import (
 	"strings"
 )
 
-type identityManager struct {
-	keystoreManager keystoreInterface
+type IdentityManager struct {
+	KeystoreManager keystoreInterface
```

Comment:
This attribute was intentionally private.
- lets force usage of factory `NewIdentityManager()`
- lets ramake to `NewIdentityManager(keydir string)` -> `NewIdentityManager(keystoryManager)`

Reply:
done